bindings/test/unit/test_auth.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+
+import mock
+
+from pulp.bindings import auth
+
+
+class TestUserCreate(unittest.TestCase):
+    def setUp(self):
+        self.server = mock.MagicMock()
+        self.api = auth.UserAPI(self.server)
+
+    def test_correct_passthrough(self):
+        self.api.create('me', 'letmein', 'Mr. Me')
+
+        self.server.POST.assert_called_once_with('/v2/users/',
+                                                 {'login': 'me',
+                                                  'password': 'letmein',
+                                                  'name': 'Mr. Me'})
```

Comment:
On 03/30/2015 03:19 PM, Austin wrote:
> We just say 'Me' -- @rbarlow <https://github.com/rbarlow>

Hahahaha. Yeah that's exactly the sort of thing I'd say.

-- 
Randy Barlow